catalog/ui/catalog-ui-search/src/main/webapp/js/model/Metacard.js

```diff
@@ -694,21 +695,71 @@ define([
         });
 
         MetaCard.SourceStatus = Backbone.AssociatedModel.extend({
-
+            defaults: {
+                count: 0,
+                elapsed: 0,
+                hits: 0,
+                id: 'undefined',
+                successful: undefined,
+                top: 0,
+                fromcache: 0,
+                cacheHasReturned: properties.isCacheDisabled,
+                cacheSuccessful: true,
+                cacheMessages: [],
```

Comment:
Need to update this to have defaults return a function since two of the attributes are arrays.